src/Microsoft.Health.Fhir.Shared.Core.UnitTests/Features/Search/SearchParameters/SearchParameterDefinitionManagerTests.cs

```diff
@@ -282,11 +282,15 @@ public async Task GivenASearchParameterDefinitionManager_WhenGettingSearchParame
                 Url = "http://test/Patient-test",
                 Type = Hl7.Fhir.Model.SearchParamType.String,
                 Base = new List<ResourceType?>() { ResourceType.Patient },
-                Expression = "expression",
+                Expression = "Patient.Name",
                 Name = "test",
                 Code = "test",
             };
 
+            _searchParameterSupportResolver
+                .IsSearchParameterSupported(Arg.Is<SearchParameterInfo>(p => p.Name == "test"))
+                .Returns((true, false));
```

Comment:
>true [](start = 26, length = 4)

Any chance you want to have tests where searchParameter is unsupported, or you prefer to cover it in e2e tests?

Reply:
As I am reusing the Parse and IsSupported logic that was already existing, just calling it in a new place, I added the E2E tests.  I think that is sufficient.